bedrock/thunderbird/templates/thunderbird/start/release.html

```diff
@@ -11,6 +11,43 @@
       <header>
         <h1>{{ self.title() }}</h1>
       </header>
+      {% if false and l10n_has_tag('thunderbird_startpage_donate_2016_06') %}
```

Comment:
The content is not visible as you seen to have left a `false` in the conditional here.